test/netrix/game/netrix_test.exs

```diff
@@ -1,88 +1,96 @@
 defmodule NetrixTestGame do
   use ExUnit.Case
 
-  alias Mix.Task
   alias Netrix.Game.Tetro
 
   describe "Tetro.to_points/1" do
     test "to_points for :i" do
-      assert Tetro.to_points(%{shape: :i}) do
-        [
-          {2, 1},
-          {2, 2},
-          {2, 3},
-          {2, 4}
-        ]
-      end
```

Comment:
I'm surprised that you can even pass a block to assert. I will look into that. In the meantime, it's important to note that these tests passed no matter what. I fixed them. :)